lib/signature.coffee

```diff
@@ -1,8 +1,18 @@
 _ = require('lodash')
 _.str = require('underscore.string')
+async = require('async')
 Parameter = require('./parameter')
 settings = require('./settings')
 parse = require('./parse')
+utils = require('./utils')
+
+isLastOne = (parameters, predicate) ->
+	index = _.findIndex(parameters, predicate)
```

Comment:
I liked the first idea, although notice `predicate` is a function, so it makes little sense to compare the last one to it.

I replaced `parameters[parameters.length - 1]` with `_.last(parameters)` and came with the following solution:

```coffee
return predicate(_.last(parameters))
```